src/ApiService/ApiService/onefuzzlib/JobOperations.cs

```diff
@@ -76,7 +88,12 @@ public class JobOperations : StatefulOrm<Job, JobState>, IJobOperations {
             await _context.Events.SendEvent(new EventJobStopped(job.JobId, job.Config, job.UserInfo, taskInfo));
         }
 
-        await Replace(job);
+        var result = await Replace(job);
 
+        if (result.IsOk) {
+            return job;
+        } else {
+            throw new Exception($"Failed to save job {job.JobId}");
```

Comment:
I understand not wanting to directly expose the error from the ORM, but shouldn't we return our OnefuzzResult type instead of throwing?

Reply:
i added the error from the orm to the exception message. 
In general i am not against the idea of returning a Result instead of of throwing but we would need to change all method to do the same to be consistent which is out of scope for the pr.